client/blocks/signup-form/index.jsx

```diff
@@ -237,6 +241,14 @@ class SignupForm extends Component {
 		}
 	};
 
+	filterEmptyFieldErrors = ( errorMessages ) => {
```

Comment:
Rather than talk about empty fields, we could talk about untouched fields. I think it signals better the intent of the function because empty but dirty fields won't be filtered out. All this to say that I believe, a better name for this function would be `filterUntouchedFieldErrors`.

Reply:
Yes great point! I definitely agree, thanks! Done! 👍